parachains/runtimes/starters/shell/src/lib.rs

```diff
@@ -14,6 +14,12 @@
 // You should have received a copy of the GNU General Public License
 // along with Cumulus.  If not, see <http://www.gnu.org/licenses/>.
 
+//! # Shell Runtime
+//!
+//! The Shell runtime defines a lightweight parachain with almost no user interface. It can be used
+//! to test parachain block inclusion and finalization on a Relay Chain and listen for a downward
```

Comment:
If you want to express here that it is used for tests, then this isn't true.

Reply:
Just state that it is a bare minimum parachain runtime + the stuff you say about its upgrade functionality.